File: dynamicDebug/src/main/java/com/cloud/dynamicdebug/MockUserStateImpl.java
package com.cloud.dynamicdebug;

import com.cloud.disk.api.user.UserState;

import javax.inject.Inject;

public class MockUserStateImpl implements UserState {
    @Inject
    public MockUserStateImpl() {
    }

    @Override
    public String getUserId() {
        return "";
    }

    @Override
    public boolean isLogin() {
        return true;
    }
}

File: dynamicDebug/src/main/java/com/cloud/dynamicdebug/MockUserStateImpl.kt
package com.cloud.dynamicdebug

import com.cloud.disk.api.user.UserState
import javax.inject.Inject

class MockUserStateImpl @Inject constructor() : UserState {
    override fun getUserId(): String {
        return ""
    }

    override fun isLogin(): Boolean {
        return true
    }
}